nixnet/system/databases.py

```diff
@@ -0,0 +1,149 @@
+
+    def __getitem__(self, index):
+        # type: (str) -> Database
+        """Return the Database object associated with the specified alias.
+
+            Args:
+                index(str): The value of the index (alias name).
+        """
+        if isinstance(index, six.string_types):
+            for alias, filepath in self.__get_database_list(''):
+                if alias == index:
+                    return self._create_item(alias, filepath)
+            else:
+                raise KeyError('Database alias %s not found in the system' % index)
+        else:
+            raise TypeError(index)
+
+    def __delitem__(self, index):
+        # type: (str) -> None
+        """Delete/Remove a database alias from the system.
+
+        This function removes the alias from NI-XNET, but does not affect the
+        database text file. It just removes the alias association to the
+        database file path.
+
+        This function is supported on Windows only, and the alias is removed
+        from Windows only (not RT targets). Use 'undeploy' to remove an alias
+        from a Real-Time (RT) target.
+
+        Args:
+            index(str): The name of the alias to delete.
+        """
+        _funcs.nxdb_remove_alias(index)
+
+    def _create_item(self, database_alias, database_filepath):
+        return Database(database_alias, database_filepath)
+
+    def keys(self):
+        """Return all keys (alias names) used in the Databases object.
+
+            Yields:
+                An iterator to all the keys in the Database object.
+        """
+        for alias, _ in self.__get_database_list(''):
+            yield alias
+
+    def values(self):
+        """Return all Database objects in the system.
+
+            Yields:
+                An iterator to all the values in the Databases object.
+        """
+        for alias, filepath in self.__get_database_list(''):
+            yield self._create_item(alias, filepath)
+
+    def items(self):
+        """Return all aliases and database objects associated with those aliases in the system.
+
+            Yields:
+                An iterator to tuple pairs of alias and database objects in the system.
+        """
+        for alias, filepath in self.__get_database_list(''):
+            yield alias, self._create_item(alias, filepath)
+
+    def add_alias(self, database_alias, database_filepath, default_baud_rate):
+        # type: (str, str, int) -> None
+        """Add a new alias with baud rate size of up to 64 bits to a database file.
+
+        NI-XNET uses alias names for database files. The alias names provide a
+        shorter name for display, allow for changes to the file system without
+        changing the application, and enable efficient deployment to LabVIEW
+        Real-Time (RT) targets.
+
+        This function is supported on Windows only. For RT targets, you can pass
+        the new alias to 'deploy' to transfer an optimized binary image of the
```

Comment:
Kill all the RT stuff since we're not really supporting those use cases with this API (atm)